.gitignore

```diff
@@ -3,7 +3,7 @@
 __pycache__/
 *.py[cod]
 *$py.class
-
+*.venv
```

Comment:
Let's remove the specifics of the environment

```suggestion
```

Reply:
removed in new commit

Reply:
new commit with fix for black formatting and also could run the tests locally to catch another minor issues. 
When installing libs for testing I got numpy 2.0 version with a older version of open-cv which did not end well so I added up also a version upper limit for numpy and open-cv proven to work.